### ISSUE TITLE
Build .5229 runtime error

### ISSUE THREAD

**Issue by AlPrunty** [wwivbbs/wwiv#1058]
OK... I can get .5226 to work, but all EXE's in the .5229 bundle are still giving me...

The program can't start because api-ms-win-core-rtlsupport-l1-2-0.dll is missing from your computer. Try reinstalling the program to fix this problem.

Something was missed in the compile?  .5228 had same issue.  Can't verify .5227

Allen

**Comment by wwiv** [comment]
Unsure.  I have an older win7 vm on the nas.  I'll copy it over and try it there

**Comment by wwiv** [comment]
It runs fine on my win10 vm

**Comment by wwiv** [comment]
vm copying slowly over the network... until then there's a vcredist package bundled with msvc 2017. I'll put it on gcs and add a URL to download it. I bet that may fix it.

**Comment by wwiv** [comment]
Try installing the visual studio 2017 redist files (which apparently are only included with visual studio, and not yet available online from any place that looks official).

I copied them out of the vc/redist folder and put them into GCS here:
https://storage.googleapis.com/build-iv/redist/vc2017/vcredist_x86.exe

Let me know if that fixes it.  I believe it should.

**Comment by wwiv** [comment]
Ok, tried this on my clean win7 image and could reproduce it. Turns out I don't need to include the vc dll's nor ucrtbase.dll to have things work.  I deleted them and everything was fine.

Also it was 2555 that worked previously AFAICT not 2556. Which makes sense since I updated the DLL's in 2556. I'll just remove them as I don't think they are really needed.  Anyone who needs to install the 2017 redist package can (and should) do so.
